src/neptune/internal/utils/__init__.py

```diff
@@ -80,6 +82,11 @@ def verify_type(var_name: str, var, expected_type: Union[type, tuple]):
         raise TypeError("{} is a stream, which does not implement read method".format(var_name))
 
 
+def verify_value(var_name: str, var: Any, expected_values: Iterable[T]) -> None:
+    if var not in expected_values:
+        raise ValueError(f"{var_name} must be one of {expected_values} (was `{var}`)")
```

Comment:
Maybe some extra help, in case the meaning of active/inactive is not clear?

```suggestion
        raise ValueError(
            f"The {var_name} argument must be one of {expected_values} (was `{var}`)."
            " For info about states, see https://docs.neptune.ai/api/sys/#state"
        )
```

Reply:
Oh right